services/nginz/integration-test/resources/oauth/ed25519_public.jwk

```diff
@@ -0,0 +1 @@
+{"kty":"OKP","crv":"Ed25519","x":"mhP-NgFw3ifIXGZqJVB0kemt9L3BtD5P8q4Gah4Iklc"}
```

Comment:
There's a `services/brig/test/resources/oauth/ed25519.jwk`, and a `services/nginz/integration-test/resources/oauth/ed25519_public.jwk`. Maybe it makes sense to move these together and use it where needed?

Reply:
Addressed in https://github.com/wireapp/wire-server/pull/2989#pullrequestreview-1338926862.